src/documents/matching.py

```diff
@@ -60,14 +60,15 @@ def match_tags(document, classifier):
 def matches(matching_model, document):
     search_kwargs = {}
 
-    document_content = document.content.lower()
-
     # Check that match is not empty
     if matching_model.match.strip() == "":
         return False
 
     if matching_model.is_insensitive:
```

Comment:
I put together an update test file: https://gist.github.com/stumpylog/2f5ac5548a783a110bf6d6a86dc1fce6

If can include that, we'll have case sensitive covered

Reply:
Nice thanks!
The tests cover not matching upper case letter patterns to lower case content in case-sensitive mode.
To be super save there should also be tests that cover upper case letter patterns matching upper case letters content.

Reply:
All added.